corehq/apps/export/models/new.py

```diff
@@ -1238,14 +1268,16 @@ class SplitUserDefinedExportColumn(ExportColumn):
     )
     user_defined_options = ListProperty()
 
-    def get_value(self, doc, base_path, transform_dates=False, **kwargs):
+    def get_value(self, domain, doc_id, doc, base_path, transform_dates=False, **kwargs):
```

Comment:
it doesn't seem like you'd need `doc_id`, but `doc` for forms is just the form data so it doesn't have a reference to the id